docs/advanced-topics/hybrid-models.md

```diff
@@ -0,0 +1,136 @@
+# Hybrid model deployment
```

Comment:
Small comment but this file might benefit from a diagram (i.e. Client | Server, and what is evaluated where),